server/webhook_worker.go

```diff
@@ -48,8 +50,46 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
 		return err
 	}
+	// We will only send webhook events if we have a connected instance.
+	instance, err := ww.p.instanceStore.LoadInstance(msg.InstanceID)
+	if err != nil {
+		// This isn't an internal server error. There's just no instance installed.
```

Comment:
I am not able to understand the comment if no instance is installed then LoadInstance will give the error then what is the need of this comment ?